### PR TITLE
Update voicerss documentazion for new options

### PR DESCRIPTION
**Pull request by nagyrobi**
## Proposed change

New options available for voicerss; clarify which are the languages and codecs supported currently by the integration (as the service may support more in the future).

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/41699 , https://github.com/home-assistant/core/pull/40502 (already in)
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/voicerss.markdown [hunk 1]
```diff
@@ -34,15 +34,30 @@ codec:
...
   type: string
-  default: mp3
+  default: " mp3 "
 format:
   description: The audio sample format.
   required: false
   type: string
-  default: 8khz_8bit_mono
+  default: " 8khz_8bit_mono "
+voice:
```
Comment (frenck) [inline source/_integrations/voicerss.markdown:43]: We no longer accept YAML configuration for integrations that connect to devices or services.

Instead, a configuration flow via the UI should be used.

For more information about this decision see:

- ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>
- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To get started on the integration configuration, see our developer documentation:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/voicerss.markdown (hunk 1 above)
Comment (nagyrobi) [reply source/_integrations/voicerss.markdown:43]: I wish there was proper documentation for to how to create a config flow, with 'Hello world' -style samples etc. The documentation linked is too vague for a beginner.

**Comment by nagyrobi** [comment]
Since I have no clue where to even start creating a config flow, I saved my current work into a custom component:
https://github.com/nagyrobi/custom_components_voicerss/
This will override the internal component with the same name so if one wants, can still use it instead.

**Comment by frenck** [comment]
Closing this PR as the parent PR has been closed.

@nagyrobi There are hundreds of examples in our codebase (search for `config_flow: true` in the integration manifests).

**Comment by nagyrobi** [comment]
> There are hundreds of examples in our codebase (search for `config_flow: true` in the integration manifests).

I already looked at some, all are pretty advanced, I've found none to be self explanatory enough. Zero comments etc.
I'm sorry I'm not able to do this unless I get some help.

**Comment by frenck** [comment]
Sorry to hear that, we have developer channels available on our Discord chat server if you have questions.

**Comment by nagyrobi** [comment]
I have some other custom components I maintain, see:
https://github.com/nagyrobi/home-assistant-custom-components-linkplay
https://github.com/nagyrobi/home-assistant-custom-components-cover-rf-time-based
But I fail converting these to the new requirements system like config_flows and asyncs - I don't even hope they will ever merge into main HA, but they still to a pretty good job in the old school way.
I don't feel like going to Discord for every little question I may have, a better "Hello world" - level developer documentation would be much more appropriate rather than having to make yourself a fool for not understanding every detail of object oriented approach.

**Comment by frenck** [comment]
> I don't feel like going to Discord for every little question I may have, a better "Hello world" - level developer documentation would be much more appropriate rather than having to make yourself a fool for not understanding every detail of object oriented approach.

Contributions to the developer documentation are appreciated as well.

There is no "making yourself a fool" here, everybody learns things at some point. Nobody was born with the knowledge. Approaching it as you do, is only blocking yourself from the learning process. Which IMHO, is the lesser smart approach in general.

**Comment by nagyrobi** [comment]
The problem with the Discord is that while I was asking, all the answers I kept getting were links to existing documentation, which, as I said, is too high for me. Nobody took the time to sit down and explain some basic concepts. The way the documentation is phrased currently, is that it assumes you already know a lot about internal structures of HA, which, to be honest, is not clear where to get from.

**Pull request closed without merging**